plotly/tools.py

```diff
@@ -1586,6 +1631,13 @@ def _flatten(array):
                                          "entered as lists or ndarrays!")
 
     @staticmethod
+    def _hex_to_rgb(value):
+        value = value.lstrip('#')
+        lv = len(value)
+        return tuple(int(value[i:i + lv // 3], 16)
+                     for i in range(0, lv, lv // 3))
```

Comment:
:cow2: I know this is a small function, but `lv` is hard to understand and there's a whole lot happening in this comprehension. Can we split `lv // 3` into a new variable, for example?